### DISCUSSION TITLE
multi-node [Jun 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2315] [Discord Threads]
> 🙋‍♂️ **not_a_bot_mkay**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### NC_REDIS_URL   
####    
#### https://docs.nocodb.com/getting-started/installation  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/981608290075164812)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
